batch/sql/increase-test-and-dev-pool-sizes.py

```diff
@@ -0,0 +1,31 @@
+import os
+import asyncio
+from gear import Database
+
+
+async def main():
```

Comment:
I wonder if this is better as a build.yaml step instead of a migration. Then we can just change it (again) if we want different settings, instead of having to apply another whole migration.

Reply:
> I could've taken a day to write and test pool APIs and create a proper PR for that

I just meant directly modify the database in a build.yaml step by mounting the database credentials.  You don't need an API or anything.